1 HW_Standard/HW_Hitmonlee.yaml

```diff
@@ -0,0 +1,21 @@
+species: Hitmonlee
+ingamename: Cain
+displayname: SCP-073
+setname: Halloween
+tags: [halloween]
+item: Metal Coat
+ability: Rough Skin
+moves:
+    - [Toxic]
+    - [Sludge Bomb]
+    - [Counter]
+    - [Mirror Coat]
+happiness: 255
+biddable: True
+rarity: 1.0
+ball: [Dusk]
+shiny: True
+nature: Serious
+ivs: {hp: 31, atk: 31, def: 31, spA: 31, spD: 31, spe: 31}
```

Comment:
sludge bomb is very weak, but i dunno how to value countercoat, I'l like a more experiance person to chip in.